tareas/clase-5/tarea-1/tarea-1.2/tarea-1.2.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Tarea 1 clase #5 1.2</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+</head>
+
+<body>
+    <h1>Tarea 1.2 clase #5</h1>
+    <p>TAREA: completar tareas/clase-5/index.html para que incluya tarea-clase-5.js</p>
+
+    <p>TAREA: crear un formulario donde un usuario pueda ingresar su salario <strong>mensual</strong></p>
+    <p>Cuando el usuario haga click en el botón "calcular", mostrar el salario <strong>anual</strong> en una caja de
+        texto deshabilitada.</p>
+    <p>En caso de no saber su salario anual ingrese su salario mensual</p>
+    <form>
+        <label for="salario-mensual">Salario Mensual:</label>
+        <input type="number" id="salario-mensual-usuario" placeholder="Ingrese su salario mensual">
```

Comment:
el "for" es para indicar que cuando le hacés click al label, tiene que hacer focus en el elemento con id "salario-mensual", así que el "for" debería ser for="salario-mensual-usuario". Se entiende? Lo mismo con los otros labels.